cardano-api-gen/src/Gen/Cardano/Api/Typed.hs

```diff
@@ -46,21 +46,20 @@ module Gen.Cardano.Api.Typed
 
 import           Cardano.Api hiding (txIns)
 import qualified Cardano.Api as Api
-import           Cardano.Api.Byron (Lovelace(Lovelace), KeyWitness(ByronKeyWitness),
-                    WitnessNetworkIdOrByronAddress(..) )
-import           Cardano.Api.Shelley (Hash(ScriptDataHash), KESPeriod(KESPeriod),
-                    StakePoolKey, PlutusScript(PlutusScriptSerialised),
-                    StakeCredential(StakeCredentialByKey),
-                    ProtocolParameters(ProtocolParameters),
-                    OperationalCertificateIssueCounter(OperationalCertificateIssueCounter) )
+import           Cardano.Api.Byron (KeyWitness (ByronKeyWitness), Lovelace (Lovelace),
+                   WitnessNetworkIdOrByronAddress (..))
+import           Cardano.Api.Shelley (Hash (ScriptDataHash), KESPeriod (KESPeriod),
+                   OperationalCertificateIssueCounter (OperationalCertificateIssueCounter),
+                   PlutusScript (PlutusScriptSerialised), ProtocolParameters (ProtocolParameters),
+                   StakeCredential (StakeCredentialByKey), StakePoolKey)
```

Comment:
There are no "real" changes in this module. Can we avoid changing the file at all?